test/test_mps.py

```diff
@@ -987,7 +989,10 @@ def mps_ops_modifier(ops):
         'nn.functional.avg_pool2d': [torch.float16],
         # input types 'tensor<f32>' and 'tensor<1xf16>' are not broadcast compatible
         # Refer to the issue please: https://github.com/pytorch/pytorch/issues/124252
-        'nn.functional.binary_cross_entropy': [torch.float16]
+        'nn.functional.binary_cross_entropy': [torch.float16],
+
+        'nn.functional.nll_loss': [torch.float16],
```

Comment:
@etaf in the future, please add a comment why are you adding a function to a skip-list (and create an issue)
I though the reason for the skip were that it crashed the runtime, but looks like it just it needed a precision tolerance tweak

Reply:
@malfet thanks for your reminder,  I'll take care of it next time。